### PR TITLE
Updates old readme and creates new one for updated codebase

### PR DESCRIPTION
**Pull request by hillarykhan**
Naming: labs36-readme-edits

Reviewers: minimum 2 (Chris and/or 2 from Web or DS respectively and not person who submitted PR)

Description: 
  What is the motivation for changes?
- As the project comes to a close, labs36 updated the readmes to reflect changes in our codebase from the last month.
  What specific changes were made?
- The repo now holds our newer codebase but keeps the old codebase in a separate folder - we updated the readme associated with that old codebase to reflect contributions we made in the last month
- Added a new readme for the new codebase - contains update links to our API, labs36 contributions, and additional steps for how to use a requirements.txt to deploy our app locally.

Submitter Checklist: 
  - [X] Small scope (1-2 components)
  - [X] Not too many concerns in a single commit
  - [X] Remove extraneous comments (code in comments), print statements, to-dos, console.logs, extra fluff
  - [X] Atomic, descriptive commit messages
  - [X] Consistent formatting

Reviewer Checklist:
  - [ ] Small scope (1-2 components)
  - [ ] Comments, print statements, to-dos, console.logs, extra fluff were removed
  - [ ] Give feedback no matter what conversations happened elsewhere, document here
  - [ ] Comment on the line of code itself (blue plus box) in files changed tab

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
 
-This project has been worked on by many lab teams over the past 9 months. One task of labs32 was to clean up the existing data pipeline from twitter. The admin will approve or reject the tweets based on whether the tweet truly represents an instance of police brutality.  Approved tweets will be displayed on the website with the existing reddit data.
+This project has been worked on by many lab teams over the past 10 months. In the final month of development, labs36 was tasked with finalizing our codebase and architecture to deploy a production-ready app. This included: automating our collection of Twitter data, deploying to AWS Bean Stalk, connecting our architecture to the backend team's architecture, labeling 5,000 tweets, retrainining our BERT model, creating performance metrics for our model, cleaning our codebase, and updating the documentation.
```
Comment (ericyeonpark) [inline README.md:5]: - AWS Bean Stalk should technically be AWS Elastic Bean Stalk
- retrainining our BERT model is a type (should be retraining)

File: README.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ In order for the app to function correctly, the user must set up their own envir
 		c. ACCESS_KEY
 		d. ACCESS_SECRET
```
Comment (ericyeonpark) [inline README.md:61]: Should we add in the consumer_key and consumer_secret? (not sure if I don't understand the technicality)
I think the bullet points arent alphabetical

**Review by ericyeonpark — APPROVED** [review]
Some minor typos but looks good!

File: README.md (hunk 1 above)
Comment (hillarykhan) [reply README.md:5]: Thank you for catching these - the corrections will be reflected in the next pull request!

**Pull request merged**